Directory.Packages.props

```
@@ -25,6 +25,27 @@
     <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="8.0.0" />
     <PackageVersion Include="System.Text.Json" Version="8.0.1" />
   </ItemGroup>
+  <ItemGroup Label="DotNet 7 Package Versions." Condition="'$(TargetFramework)' == '$(OldestSupportedDotNetVersion)'">

```

This condition probably needs to be extended to ensure they are not the same.
```suggestion
  <ItemGroup Label="Previous dotnet Package Versions." Condition="'$(TargetFramework)' == '$(OldestSupportedDotNetVersion)' And '$(OldestSupportedDotNetVersion)' != '$(LatestSupportedDotNetVersion)'">

```